Dockerfile

```diff
@@ -49,13 +49,12 @@ RUN addgroup \
   echo "git:$(dd if=/dev/urandom bs=24 count=1 status=none | base64)" | chpasswd
 
 ENV USER git
-ENV GITEA_CUSTOM /data/gitea
 
 VOLUME ["/data"]
 
 ENTRYPOINT ["/usr/bin/entrypoint"]
 CMD ["/bin/s6-svscan", "/etc/s6"]
 
 COPY docker /
-COPY --from=build-env /go/src/code.gitea.io/gitea/gitea /app/gitea/gitea
+COPY --from=build-env /go/src/code.gitea.io/gitea/gitea-docker /app/gitea/gitea
```

Comment:
This binary is generate in the previous stage of the dockerfile not from drone: https://github.com/go-gitea/gitea/pull/6631/files#diff-3254677a7917c6c01f55212f86c57fbfR19
That what is blocking here.

Personally, I don't think we need a special binary for docker setting env variable doesn't add overweight on image and it how the user of the container will configure the running container

Reply:
OK I just wanted to show how we could simplify our docker environment and to show that this works. Without the changes to the docker build I think we might need to add some tests to ensure that this doesn't get broken by mistake.

Reply:
OK I've removed those changes. 

I do think it would be easier if the `gitea` installed on the docker was just built to work with it's paths correct for the default docker installation without environment variables or options etc. but it could be done at a later date.